File: source/code/csharp/authentication/saml_policy_example.cs
var samlPolicy = await app.GetSamlPolicyAsync();

var samlServiceProvider = await samlPolicy.GetSamlServiceProviderAsync();
// samlServiceProvider.Href is the SAML Service Provider URL

var ssoInitiationEndpoint = await samlServiceProvider.GetSsoInitiationEndpointAsync();
// ssoInitiationEndpoint.Href is the SSO Initiation Endpoint

File: source/code/java/authentication/saml_policy_example.java
SamlPolicy policy = application.getSamlPolicy();

SamlServiceProvider servicePovider = policy.getSamlServiceProvider();
// servicePovider.getHref() is the SAML Service Provider URL

SsoInitiationEndpoint ssoInitiationEndpoint = servicePovider.getSsoInitiationEndpoint();
// ssoInitiationEndpoint.getHref() is the SSO Initiation Endpoint
